### ISSUE TITLE
Return value not correct when a vertex has same name in multiple tags

### ISSUE THREAD

**Issue by jude-zhu** [vesoft-inc/nebula#581]
First, I create three tags and an edge
```
CREATE TAG course(name string, credits int)
CREATE TAG building(name string)
CREATE TAG student(name string, age int, gender string)
CREATE EDGE select(grade int)
```
Then, insert the following data
```
INSERT VERTEX course(name, credits),building(name) VALUES 102:("English", 6, "No11");
INSERT VERTEX student(name, age, gender) VALUES 202:("Jane", 17, "female");
INSERT EDGE select (grade) VALUES 202 -> 102:(3);
```
when query
```
GO FROM 202 OVER select YIELD $^[student].name AS Student, $$[course].name AS Curriculum, select.grade AS Grade, $$[building].name AS location
```

return unexpected Curriculum
![1](https://user-images.githubusercontent.com/51590253/60639750-865b0980-9e56-11e9-962a-7385696e4286.jpg)

**Comment by whitewum** [comment]
can u paste ur pic? a bit hard to follow
anyway. great

**Comment by laura-ding** [comment]
UTs has cover multi-tag of a vertex, but prop name of tags is diff, so the bug can’t test out. But here diff tags in the vertex has same prop name .so it happens. Thanks for you@jude-zhu

**Comment by sherman-the-tank** [comment]
Good catch 👍 

We need a UT to cover this case
